presto-main/src/test/java/io/prestosql/execution/resourcegroups/TestResourceGroups.java

```diff
@@ -113,29 +112,26 @@ public void testFairEligibility()
         assertEquals(query3a.getState(), QUEUED);
 
         query1a.complete();
-        root.updateGroupsAndProcessQueuedQueries();
```

Comment:
Note that these are not needed. query1a.complete() will automatically trigger finding and running the next query.